Add the Repositories and repair the content for RHEL 8 (#1279)
theforeman/foreman-documentation: guides/common/modules/proc_attaching-satellite-infrastructure-subscription.adoc

= Attaching the {Project} Infrastructure Subscription

After you have registered {ProductName}, you must identify your subscription Pool ID and attach an available subscription.

> The {ProjectName} Infrastructure subscription provides access to the {ProjectName} and {RHEL} content. For {RHEL} 7, you may require Red Hat Software Collections (RHSCL). This is the only subscription required.

Reviewer: ```suggestion
The {ProjectName} Infrastructure subscription provides access to the {ProjectName} and {RHEL} content. For {RHEL} 7 it also provides access to Red{nbsp}Hat Software Collections (RHSCL). This is the only subscription required.
```

Author: Modified.
